definitions/tools/pvacbind.cwl

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "run pVACseq"
+
+baseCommand: [
+    "ln", "-s"
+]
+arguments: [
+    { valueFrom: "$TMPDIR", shellQuote: false },
+    "/tmp/pvacbind",
+    { valueFrom: " && ", shellQuote: false },
+    "export", "TMPDIR=/tmp/pvacbind",
+    { valueFrom: " && ", shellQuote: false },
+    "/opt/conda/bin/pvacbind", "run",
+    "--iedb-install-directory", "/opt/iedb",
+    { position: 5, valueFrom: $(runtime.outdir) },
+]
+requirements:
+    - class: ShellCommandRequirement
+    - class: DockerRequirement
+      dockerPull: "griffithlab/pvactools:1.5.0"
+    - class: ResourceRequirement
+      ramMin: 16000
+      coresMin: 8
```

Comment:
I'd either wire this up to the input:
```suggestion
      coresMin: $(inputs.n_threads)
```
or convert the `n_threads` input into an argument.